src/qt/rpcconsole.cpp

```diff
@@ -9,25 +9,24 @@
 #include <qt/rpcconsole.h>
 #include <qt/forms/ui_debugwindow.h>
 
-#include <qt/bantablemodel.h>
-#include <qt/clientmodel.h>
-#include <qt/platformstyle.h>
-#include <qt/walletmodel.h>
 #include <chainparams.h>
 #include <interfaces/node.h>
 #include <netbase.h>
-#include <rpc/server.h>
+#include <qt/bantablemodel.h>
+#include <qt/clientmodel.h>
+#include <qt/platformstyle.h>
 #include <rpc/client.h>
+#include <rpc/server.h>
+#include <univalue.h>
 #include <util/system.h>
 
 #include <openssl/crypto.h>
 
-#include <univalue.h>
```

Comment:
univalue is a library, so shouldn't move that?

Reply:
@MarcoFalke Sure!